### PR TITLE
Add thread for PathoGFAIR preprint

### PR DESCRIPTION
**Pull request by bebatut**
Hi,

I hope everything is okay (my first try of the tool there)

Bérénice

### CODE REVIEW

**Comment by arash77** [comment]
Media `bluesky` and `mastodon` are disabled in `plugins.yml`. either you have to delete them from your post file or, we have to add the credentials first and enable them.

File: posts/2024-07-09-pathogfair-preprint.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
+---
+media:
+  - bluesky
+  - mastodon
```
Comment (bgruening) [inline posts/2024-07-09-pathogfair-preprint.md:4]:
```suggestion
```

File: posts/2024-07-09-pathogfair-preprint.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+  - mastodon-eu-freiburg
+  - matrix-eu-announce
+
+mentions:
+  bluesky:
+    - bebatut.bsky.social
+  mastodon:
+    - bebatut@piaille.fr
```
Comment (bgruening) [inline posts/2024-07-09-pathogfair-preprint.md:12]:
```suggestion
```

File: posts/2024-07-09-pathogfair-preprint.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+
+hashtags:
+  bluesky:
+    - metagenomics
+    - pathogen
+    - usegalaxy
+  mastodon:
+    - metagenomics
+    - pathogen
+    - usegalaxy
```
Comment (bgruening) [inline posts/2024-07-09-pathogfair-preprint.md:24]:
```suggestion
```

File: posts/2024-07-09-pathogfair-preprint.md (hunk 1 above)
Comment (bgruening) [inline posts/2024-07-09-pathogfair-preprint.md:4]:
```suggestion
```

File: posts/2024-07-09-pathogfair-preprint.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+
+hashtags:
+  mastodon-eu-freiburg:
+    - metagenomics
+    - pathogen
+    - usegalaxy
+
+---
+
+**1/6 🚀 Exciting news in pathogen detection!**
```
Comment (bgruening) [inline posts/2024-07-09-pathogfair-preprint.md:18]:
```suggestion
**🚀 Exciting news in pathogen detection!**  
```

As you see in the preview we add the 1/6 automatically in medias where is makes sense. Maston we add 1/6 but on Matrix we don't.
Does that make sense?

File: posts/2024-07-09-pathogfair-preprint.md [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+---
+
+**1/6 🚀 Exciting news in pathogen detection!**  
+Our new PathoGFAIR preprint introduces FAIR and adaptable (meta)genomics workflows for detecting and tracking foodborne pathogens. This project started nearly three years ago as a collaboration with Biolytix, a swiss SME. 🌟
+
+🔗 [Preprint](https://www.biorxiv.org/content/10.1101/2024.06.26.600753v1)  
+🔗 [Blogpost](https://galaxyproject.org/news/2024-07-08-pathogfair-preprint/)
+
+
+**2/6 🏁 Project Origins**
```
Comment (bebatut) [inline posts/2024-07-09-pathogfair-preprint.md:25]:
```suggestion
**🏁 Project Origins**  
```

File: posts/2024-07-09-pathogfair-preprint.md [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+🔗 [Blogpost](https://galaxyproject.org/news/2024-07-08-pathogfair-preprint/)
+
+
+**2/6 🏁 Project Origins**  
+The journey began with funding from EOSC-Life for Industry Collaboration to modernize and validate Biolytix's bioinformatic pipelines. The goal? To make pathogen detection workflows accessible and scalable via the Galaxy platform. 🌐
+
+🔗 [Project Background](https://galaxyproject.org/news/2021-12-08-pathogen-detection-eosc-life-grant/)
+
+
+**3/6 🔍 What's in PathoGFAIR?**
```
Comment (bebatut) [inline posts/2024-07-09-pathogfair-preprint.md:31]:
```suggestion
**🔍 What's in PathoGFAIR?**  
```

File: posts/2024-07-09-pathogfair-preprint.md [hunk 7]
```diff
@@ -0,0 +1,58 @@
...
+- **Preprocessing**
+- **Taxonomy Profiling and Visualization**
+- **Gene-based Pathogen Identification** with assembly, AMR and VF gene detection
+- **Allele-based Pathogen Identification** with variant calling
+- **PathoGFAIR Samples Aggregation and Visualisation** including phylogenetic analysis
+
+🔗 [Workflows](https://usegalaxy-eu.github.io/PathoGFAIR/#how-to-find-pathogfair-workflows)
+
+
+**4/6 🌟 Embracing FAIR Principles**
```
Comment (bebatut) [inline posts/2024-07-09-pathogfair-preprint.md:42]:
```suggestion
**🌟 Embracing FAIR Principles**  
```

File: posts/2024-07-09-pathogfair-preprint.md [hunk 8]
```diff
@@ -0,0 +1,58 @@
...
+🔗 [Workflows](https://usegalaxy-eu.github.io/PathoGFAIR/#how-to-find-pathogfair-workflows)
+
+
+**4/6 🌟 Embracing FAIR Principles**  
+These workflows adhere to FAIR (Findable, Accessible, Interoperable, Reusable) principles, ensuring they are easy to find (via WorkflowHub and Dockstore), share, adapt, and integrate with other data and tools. This enhances collaboration and transparency in pathogen research.
+
+![graphical abstract](./images/2024-07-09-pathogair-preprint-graphical-abstract.png)
+
+
+**5/6 📚 Training Material Available**
```
Comment (bebatut) [inline posts/2024-07-09-pathogfair-preprint.md:48]:
```suggestion
**📚 Training Material Available**  
```

File: posts/2024-07-09-pathogfair-preprint.md [hunk 9]
```diff
@@ -0,0 +1,58 @@
...
+![graphical abstract](./images/2024-07-09-pathogair-preprint-graphical-abstract.png)
+
+
+**5/6 📚 Training Material Available**  
+To support users, a comprehensive tutorial have been developed. These resources guide users through the workflows, ensuring they can effectively utilize the tools for their research and public health initiatives.
+
+🔗 [Tutorial](https://training.galaxyproject.org/training-material/topics/microbiome/tutorials/pathogen-detection-from-nanopore-foodborne-data/tutorial.html)
+
+
+**6/6 🌍 Impact on Global Health**
```
Comment (bebatut) [inline posts/2024-07-09-pathogfair-preprint.md:54]:
```suggestion
**🌍 Impact on Global Health**  
```

**Comment by bebatut** [comment]
Is it normal that the graphical abstract does not render?

**Comment by arash77** [comment]
Right now Mastodon and Bluesky do not support markdown syntax (we only support images).
I will fix this to just ignore any markdown for these social media platforms.
So it is better right now to not use markdown syntax.

**Comment by arash77** [comment]
> Is it normal that the graphical abstract does not render?

Good point!
So right now the images will be separated from the file and parsed just into the first post. But I am not sure why it is not working right now.

File: posts/2024-07-09-pathogfair-preprint.md [hunk 10]
```diff
@@ -0,0 +1,58 @@
...
+- **Allele-based Pathogen Identification** with variant calling
+- **PathoGFAIR Samples Aggregation and Visualisation** including phylogenetic analysis
+
+🔗 [Workflows](https://usegalaxy-eu.github.io/PathoGFAIR/#how-to-find-pathogfair-workflows)
+
+
+**🌟 Embracing FAIR Principles**  
+These workflows adhere to FAIR (Findable, Accessible, Interoperable, Reusable) principles, ensuring they are easy to find (via WorkflowHub and Dockstore), share, adapt, and integrate with other data and tools. This enhances collaboration and transparency in pathogen research.
+
+![graphical abstract](./images/2024-07-09-pathogair-preprint-graphical-abstract.png)
```
Comment (arash77) [inline posts/2024-07-09-pathogfair-preprint.md:45]:
```suggestion
![graphical abstract](https://raw.githubusercontent.com/usegalaxy-eu/galaxy-social/335ea9443332660a9f406c2439ff95e832426eee/posts/images/2024-07-09-pathogair-preprint-graphical-abstract.png)
```

**Comment by bebatut** [comment]
That looks good :)

**Comment by bebatut** [comment]
How should we do then?
Should I remove all Markdown formatting then? It might be good to explain that in the README

**Comment by arash77** [comment]
> How should we do then? Should I remove all Markdown formatting then? It might be good to explain that in the README

For now yes but I will fix this issue.

It is mentioned in the README file:
> Note that some social media platforms do not support Markdown formatting, so use plain text only.

**Comment by bebatut** [comment]
> It is mentioned in the README file:
> 
> > Note that some social media platforms do not support Markdown formatting, so use plain text only.

Sorry I missed that

**Comment by arash77** [comment]
> > It is mentioned in the README file:
> > > Note that some social media platforms do not support Markdown formatting, so use plain text only.
> 
> Sorry I missed that

No worries!

File: posts/2024-07-09-pathogfair-preprint.md [hunk 11]
```diff
@@ -0,0 +1,58 @@
...
+🔗 Project Background: https://galaxyproject.org/news/2021-12-08-pathogen-detection-eosc-life-grant/
+
+
+🔍 What's in PathoGFAIR?
+PathoGFAIR consists of 5 workflows:
+- **Preprocessing**
+- **Taxonomy Profiling and Visualization**
+- **Gene-based Pathogen Identification** with assembly, AMR and VF gene detection
+- **Allele-based Pathogen Identification** with variant calling
+- **PathoGFAIR Samples Aggregation and Visualisation** including phylogenetic analysis
```
Comment (arash77) [inline posts/2024-07-09-pathogfair-preprint.md:37]:
```suggestion
- Preprocessing
- Taxonomy Profiling and Visualization
- Gene-based Pathogen Identification with assembly, AMR and VF gene detection
- Allele-based Pathogen Identification with variant calling
- PathoGFAIR Samples Aggregation and Visualisation including phylogenetic analysis
```

**Comment by arash77** [comment]
Great.
Sorry about that. I will fix this as soon as possible to ignore the markdown on Mastodon and Bluesky.

File: posts/2024-07-09-pathogfair-preprint.md [hunk 12]
```diff
@@ -0,0 +1,58 @@
...
+
+
+🌟 Embracing FAIR Principles
+These workflows adhere to FAIR (Findable, Accessible, Interoperable, Reusable) principles, ensuring they are easy to find (via WorkflowHub and Dockstore), share, adapt, and integrate with other data and tools. This enhances collaboration and transparency in pathogen research.
+
+![graphical abstract](https://raw.githubusercontent.com/usegalaxy-eu/galaxy-social/335ea9443332660a9f406c2439ff95e832426eee/posts/images/2024-07-09-pathogair-preprint-graphical-abstract.png)
+
+
+📚 Training Material Available
+To support users, a comprehensive tutorial have been developed. It guides users through the workflows, ensuring they can effectively utilize the tools for their research and public health initiatives.
```
Comment (wm75) [inline posts/2024-07-09-pathogfair-preprint.md:49]:
```suggestion
To support users, a comprehensive tutorial has been developed. It guides users through the workflows, ensuring they can effectively utilize the tools for their research and public health initiatives.
```

File: posts/2024-07-09-pathogfair-preprint.md [hunk 13]
```diff
@@ -0,0 +1,58 @@
...
+
+mentions:
+  mastodon-eu-freiburg:
+    - bebatut@piaille.fr 
+
+hashtags:
+  mastodon-eu-freiburg:
+    - metagenomics
+    - pathogen
+    - usegalaxy
```
Comment (wm75) [inline posts/2024-07-09-pathogfair-preprint.md:14]:
```suggestion
    - UseGalaxy
```

**Review by wm75 — COMMENTED** [review]
You could also now bring the markdown syntax back if you want to since @arash77 has addressed this in #58, i.e. your post would now use the markdown on matrix but use plain text on mastodon. You don't have to, of course - also looks good without it.

**Comment by bebatut** [comment]
> You could also now bring the markdown syntax back if you want to since @arash77 has addressed this in #58, i.e. your post would now use the markdown on matrix but use plain text on mastodon. You don't have to, of course - also looks good without it.

Tbh I am lazy to put the Md syntax back 😅

File: posts/2024-07-09-pathogfair-preprint.md [hunk 14]
```diff
@@ -0,0 +1,58 @@
...
+To support users, a comprehensive tutorial has been developed. It guides users through the workflows, ensuring they can effectively utilize the tools for their research and public health initiatives.
+
+🔗 Tutorial: https://training.galaxyproject.org/training-material/topics/microbiome/tutorials/pathogen-detection-from-nanopore-foodborne-data/tutorial.html
+
+
+🌍 Impact
+PathoGFAIR is set to improve pathogen tracking, providing robust tools for researchers and public health officials worldwide. By improving detection capabilities, we can better protect public health. 🌎🔬
+
+📝 Preprint: https://www.biorxiv.org/content/10.1101/2024.06.26.600753v1
+🔗 Blogpost: https://galaxyproject.org/news/2024-07-08-pathogfair-preprint/
```
Comment (arash77) [inline posts/2024-07-09-pathogfair-preprint.md:58]:
```suggestion
**🚀 Exciting news in pathogen detection!**
Our new PathoGFAIR preprint introduces FAIR and adaptable (meta)genomics workflows for detecting and tracking foodborne pathogens. This project started nearly three years ago as a collaboration with Biolytix, a swiss SME. 🌟

📝 [Preprint](https://www.biorxiv.org/content/10.1101/2024.06.26.600753v1)
🔗 [Blogpost](https://galaxyproject.org/news/2024-07-08-pathogfair-preprint/)


**🏁 Project Origins**
The journey began with funding from EOSC-Life for Industry Collaboration to modernize and validate Biolytix's bioinformatic pipelines. The goal? To make pathogen detection workflows accessible and scalable via the Galaxy platform. 🌐

🔗 [Project Background](https://galaxyproject.org/news/2021-12-08-pathogen-detection-eosc-life-grant/)


**🔍 What's in PathoGFAIR?**
PathoGFAIR consists of 5 workflows:
- **Preprocessing**
- **Taxonomy Profiling and Visualization**
- **Gene-based Pathogen Identification** with assembly, AMR and VF gene detection
- **Allele-based Pathogen Identification** with variant calling
- **PathoGFAIR Samples Aggregation and Visualisation** including phylogenetic analysis

🔗 [Workflows](https://usegalaxy-eu.github.io/PathoGFAIR/#how-to-find-pathogfair-workflows)


**🌟 Embracing FAIR Principles**
These workflows adhere to FAIR (Findable, Accessible, Interoperable, Reusable) principles, ensuring they are easy to find (via WorkflowHub and Dockstore), share, adapt, and integrate with other data and tools. This enhances collaboration and transparency in pathogen research.

![graphical abstract](https://raw.githubusercontent.com/usegalaxy-eu/galaxy-social/335ea9443332660a9f406c2439ff95e832426eee/posts/images/2024-07-09-pathogair-preprint-graphical-abstract.png)


**📚 Training Material Available**
To support users, a comprehensive tutorial has been developed. It guides users through the workflows, ensuring they can effectively utilize the tools for their research and public health initiatives.

🔗 [Tutorial](https://training.galaxyproject.org/training-material/topics/microbiome/tutorials/pathogen-detection-from-nanopore-foodborne-data/tutorial.html)


**🌍 Impact**
PathoGFAIR is set to improve pathogen tracking, providing robust tools for researchers and public health officials worldwide. By improving detection capabilities, we can better protect public health. 🌎🔬

📝 [Preprint](https://www.biorxiv.org/content/10.1101/2024.06.26.600753v1)
🔗 [Blogpost]( https://galaxyproject.org/news/2024-07-08-pathogfair-preprint/)
```

**Comment by bgruening** [comment]
I merged main into this branch, maybe we get LinkedIn working as well for this post already?

**Comment by bebatut** [comment]
> I merged main into this branch, maybe we get LinkedIn working as well for this post already?

LinkedIn works now?

**Comment by bgruening** [comment]
It should yes. We need to add it to the metadata in your post I think. 

ping @arash77

File: posts/2024-07-09-pathogfair-preprint.md [hunk 15]
```diff
@@ -0,0 +1,58 @@
+---
+media:
+  - mastodon-eu-freiburg
+  - matrix-eu-announce
```
Comment (bgruening) [inline posts/2024-07-09-pathogfair-preprint.md:4]:
```suggestion
  - matrix-eu-announce
  - linkedin-galaxyproject
```

It that correct @arash77

**Comment by arash77** [comment]
#65 should get merged before that.

**Comment by bebatut** [comment]
Closed to get a cleaner PR that could be a good showcase to link to

**Pull request closed without merging**